File: dart/beginner/1182.dart
import 'dart:io';

void main() {
  List<List<double>> matriz =
      List.generate(12, (index) => List.filled(12, 0.0));
  double total = 0;
  int C = 0;
  String T = 'S';

  C = int.parse(stdin.readLineSync().toString().trim());
  T = stdin.readLineSync().toString().trim();

  for (int i = 0; i < 12; i++) {
    for (int j = 0; j < 12; j++) {
      matriz[i][j] = double.parse(stdin.readLineSync().toString().trim());
    }
  }

  matriz.forEach((row) {
    total += row[C];
  });

  if (T == 'M') total /= 12;

  print(total.toStringAsFixed(1));
}

File: C/beginner/1182.c
#include <stdio.h>

int main(){

  double matriz[12][12];
  double total = 0;
  int C;
  char T;

  scanf("%d", &C);
  scanf("%c", &T);

  scanf("%c"); // Cleaning Buffer

  for(int i = 0; i < 12; i++) 
    for(int j = 0; j < 12; j++) 
      scanf("%lf", &matriz[i][j]);
    
  for(int i = 0; i < 12; i++) total += matriz[i][C];
  if(T == 'M') total /= 12;

  printf("%.1lf\n", total);

  return 0;
}